rmgpy/chemkin.pyx

```diff
@@ -1035,14 +1035,16 @@ def load_chemkin_file(path, dictionary_path=None, transport_path=None, read_comm
     # Read in the thermo data from the thermo file        
     if thermo_path:
         with open(thermo_path, 'r') as f:
+            previous_line = f.tell()
             line0 = f.readline()
             while line0 != '':
```

Comment:
I think you could safely do this:
```suggestion
            line0 = None
            while line0 != '":
                previous_line = f.tell()
                line0 = f.readline()
```
and then delete the duplicated `previous_line` and `line0` lines at the bottom of the `while` block, but double check me on this.

Reply:
Yes, this works. I added it in and tested it.